examples/go.mod

```
@@ -0,0 +1,19 @@
+module go.mongodb.org/atlas-sdk/v20231115008/examples
+
+go 1.20
```

latest support go is 
```suggestion
go 1.21
```
if you go above 1.20 you now have to define a full version like 1.21.1